bob-apps/bin/bobd.rs

```diff
@@ -191,5 +235,12 @@ fn get_matches<'a>() -> ArgMatches<'a> {
                 .long("port")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("init folders")
```

Comment:
It is better to use the same name as specified in `long` section (`init_folders`)